volume/drivers/extpoint.go

```diff
@@ -43,18 +44,20 @@ type volumeDriver interface {
 
 // Store is an in-memory store for volume drivers
 type Store struct {
-	extensions   map[string]volume.Driver
-	mu           sync.Mutex
-	driverLock   *locker.Locker
-	pluginGetter getter.PluginGetter
+	extensions       map[string]volume.Driver
+	mu               sync.Mutex
+	driverLock       *locker.Locker
+	pluginGetter     getter.PluginGetter
+	pluginAPITimeout time.Duration
 }
 
 // NewStore creates a new volume driver store
-func NewStore(pg getter.PluginGetter) *Store {
+func NewStore(pg getter.PluginGetter, timeout time.Duration) *Store {
```

Comment:
Just a quick thought; not sure if more parameters will be added at some point; perhaps an option type, or [functional options](https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis) would be an idea.

Reply:
Thats a good point, we can wrap it inside a more generic.